### PR TITLE
[DNM] Cryo on core maps [TESTMERGE ONLY]

### PR DESCRIPTION
**Pull request by KathrinBailey**
## About The Pull Request
• Adds cryo on Metastation, Boxstation, Deltastation & Pubbystation.
• Reworks the south of medical on Metastation as little showcase of Cobalt's aesthetic changes. Adds cloning.
• A temporary addition of a small cloning room to Deltastation.

## Why It's Good For The Game
This is solely for the rebase tests only. As we do not yet have the infrastructure to modularise maps, these change are to core maps. Please only test-merge.

## Changelog
:cl:
add: Metastation: Cryo, cloning & patient rooms.
add: TEMPORARY: Cryo and cloning for Deltastation. (rebase test only)
add: TEMPORARY: Cryo for Pubbystation and Icebox. (rebase test only)
/:cl:

image
image

### CODE REVIEW

**Comment by ShadeAware** [comment]
Very pretty 👍

File: _maps/map_files/KiloStation/KiloStation.dmm [hunk 1]
```diff
@@ -23457,8 +23467,8 @@
 /area/engine/atmos)
 "aMM" = (
 /obj/machinery/door/airlock/maintenance{
-	name = "security maintenance";
-	req_access_txt = "12;63"
+	name = "Blueshield Office Maintenance";
+	req_access_txt = "71"
```
Comment (atakiya) [inline _maps/map_files/KiloStation/KiloStation.dmm]:
```suggestion
	req_access_txt = "73"
```
New blueshield access is `73` due to #2448

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 2]
```diff
@@ -72326,6 +72296,17 @@
 	},
 /turf/open/floor/wood,
 /area/crew_quarters/theatre)
+"ojw" = (
+/obj/machinery/door/airlock/maintenance{
+	name = "Blueshield Office Maintenance";
+	req_access_txt = "71"
```
Comment (atakiya) [inline _maps/map_files/MetaStation/MetaStation.dmm]: ditto
```suggestion
	req_access_txt = "73"
```

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 3]
```diff
@@ -72742,12 +72692,28 @@
...
-/obj/structure/cable,
-/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer4{
-	dir = 9
+/obj/machinery/button/door{
+	id = "psy1";
+	name = "Bolt Control";
+	normaldoorcontrol = 1;
+	pixel_x = -6;
+	pixel_y = -25;
+	req_access_txt = "71";
```
Comment (atakiya) [inline _maps/map_files/MetaStation/MetaStation.dmm:72701]:
```suggestion
	req_access_txt = "70";
```
This should be `70`
`70` is psych, `71` is science-toxins

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 4]
```diff
@@ -72742,12 +72692,28 @@
...
+	specialfunctions = 4
 	},
-/turf/open/floor/plating,
-/area/maintenance/aft/secondary)
+/obj/machinery/button/door{
+	id = "psyshu";
+	name = "Shutter button";
+	pixel_x = 6;
+	pixel_y = -25;
+	req_access_txt = "71"
```
Comment (atakiya) [inline _maps/map_files/MetaStation/MetaStation.dmm:72709]:
```suggestion
	req_access_txt = "70"
```

**Comment by Gandalf2k15** [comment]
Modular maps are now in!

**Pull request closed without merging**